### PR TITLE
Add Style Setting (Part 4)

### PR DESCRIPTION
**Pull request by theck13**
### Fix
This is the fourth part in a series of pull requests to add a style setting to the app.  The pull requests are split into parts in an attempt to make the changes smaller.  This part adds the ***Mono*** style.  See the screenshots below for illustration.

add_style_setting_all_04_

#### Note
Contact me for an installable build.

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item in drawer.
3. Tap ***Style*** item under ***Appearance*** section.
4. Notice ***Style*** screen with ***Default***, ***Mono***, ***Sepia***, ***Black***, and ***Classic*** styles.
5. Tap ***Mono*** style.
6. Notice ***Mono*** style is selected and app has ***Mono*** styling.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.

### CODE REVIEW

**Review by AmandaRiu — COMMENTED** [review]
@theck13 Code changes look good, but I did notice that the action button in this dialog was not using the correct color for dark mode:



I was using a Pixel 3 XL emulator running Android 10 for testing. 

Note that this isn't the case for dialogs in general as this dialog shows the correct font color for the action buttons:

**Comment by theck13** [comment]
@AmandaRiu, I can't reproduce that behavior with the **_Theme_** dialog in **_Dark_** mode.  The button is still a little dark, but it is using the primary color for **_Mono_**.  See the screenshots below for illustration.

1127_add_app_style_mono

I can't quite tell from your screenshot, but it appears like the selected **_Style_** is **_Black_** based on the **_Theme_** dialog.  Can you verify **_Mono_** was selected when you took that screenshot?

It's not quite the right color, but it's close.  I think that's actually a bug in the Material Components library.  If I change the dialog `colorAccent` to `red` (`#d63638`), the dialog `colorPrimary` to `blue` (`#3361cc`), and the button `textColor` to `yellow` (`#f0c930`), the dialog is styled as expected in **_Light_** mode, but not in **_Dark_**.  See the screenshots below for illustration.

1127_add_app_style_dialog

**Comment by theck13** [comment]
Actually, it wasn't a bug with the Material Components library.  The `Base.Theme.Simplestyle` style in the `values-night/styles.xml` file didn't have the `android:alertDialogTheme` and `alertDialogTheme` attributes.  That means the dialogs were using the default style.  I added those attributes in 2aaf6787.  Now, the dialogs look correct.  See the screenshots below for illustration.

1127_add_app_style_night

**Review by AmandaRiu — APPROVED** [review]
@theck13 That fixed it! :shipit:

**Comment by AmandaRiu** [comment]
Waiting for design review before merging (for reelz this time 😉 )

**Comment by SylvesterWilmott** [comment]
Looks great! Good catch on the rogue button.

**Review by SylvesterWilmott — APPROVED** [review]

**Pull request merged**
